content/docs/dvclive/api-reference/next_step.md

```diff
@@ -41,13 +41,15 @@ The _metrics summary_ will be saved to `{path}.json`. Here's an example:
 ### DVC Integration
 
 When `dvclive` is used alongside `DVC`, each `dvclive.next_step()` call will
-have additional features.
+have additional features. See
+[DVCLive with DVC](/doc/dvclive/user-guide/dvclive-with-dvc).
```

Comment:
https://dvc.org/doc/dvclive/user-guide/dvclive-with-dvc doesn't explain anything about `dvclive.next_step()`. It's just used in a code block. Maybe avoid this change  for now since it's out of scope anyway.